src/components/WorkOrder/MobileWorkingWorkOrder.js

```diff
@@ -104,8 +105,28 @@ const MobileWorkingWorkOrder = ({
             {workOrder.paymentType === OVERTIME_PAYMENT_TYPE && (
               <h4 className="lbh-heading-h4">Overtime work order</h4>
             )}
+            {operativesCount > 1 && (
+              <>
+                <h4 className="lbh-heading-h4">Operatives</h4>
+                <ol className="govuk-list govuk-!-margin-top-3">
+                  {workOrder.operatives.map((operative) => {
+                    const percentageDisplay = operative.jobPercentage
+                      ? `${operative.jobPercentage}%`
+                      : 'N/A'
+                    const operativeDisplay = [
+                      operative.name,
+                      percentageDisplay,
+                    ].join(' - ')
+                    return <li className="govuk-list">{operativeDisplay}</li>
+                  })}
+                </ol>
+              </>
+            )}
```

Comment:
Can we use the existing `<OperativeList>` component here? It accepts a `readOnly` prop which lists operatives without links and seems very close to what we want, with the exception of some margins. If the new Figma design and the existing non-readonly list differ, it's probably worth having a call with the design team to get consistency between the two kinds of lists - then we can use the same component.